src/components/molecules/UiFormField/UiFormField.vue

```diff
@@ -100,6 +120,14 @@ export interface FormFieldProps {
    * Use this props to set label hint like "Required" or "Optional"
    */
   hint?: boolean | string;
+  /**
+   * Use this props to pass value of input.
+   */
+  value?: string;
```

Comment:
![image](https://github.com/infermedica/component-library/assets/37744371/7becb9cb-d150-4a4f-b36b-8b51a478d84e)
and also `characterCounterAttrs`